modules/ROOT/partials/release-notes/6.0-release-notes-summary.adoc

```diff
@@ -27,13 +25,23 @@ This section also includes four tables that set out the:
 NOTE: In particular, this means {productname} 6.0 does not support Microsoft Internet Explorer 11.
 +
 . Premium Plugins released with {productname} 6.0 are not backwards compatible with earlier versions of {productname}.
-. {productname} 6.0 is now available in three further languages:
+. Core {productname} 6.0 is now licensed under the MIT License.
+. Three additional professionally translated language packs are now available for use with {productname} 6.0:
 * Hindi (hi);
 * Malay (ms); and
 * Vietnamese (vi)
 
+These packs are available to all https://tiny.cloud/get-tiny/[Cloud subscribers] and all https://tiny.cloud/pricing/[Premium plan subscribers].
```

Comment:
This isn't rendered as part of the above list so reads poorly. It needs a `+` on the line above or something to fix that (not sure exactly since it's 2 lists).

- [ ] Fix rendering issues

Reply:
Done - Luke made a suggestion below that was accepted